macros/sql/pivot.sql

```diff
@@ -69,7 +69,7 @@ Arguments:
     {{ agg }}(
       {% if distinct %} distinct {% endif %}
       case
-      when {{ column }} {{ cmp }} '{{ v }}'
+      when {{ column }} {{ cmp }} '{{ dbt_utils.escape_single_quotes(v) }}'
```

Comment:
not blocking but while we're in here may want to rename 'v' to something more comprehensible